>From b157e8e7569bbb467530d57271e8fe8dc5c4387f Mon Sep 17 00:00:00 2001
From: Bob Ham <rah@settrans.net>
Date: Sun, 6 Nov 2016 14:33:42 +0000
Subject: [PATCH] include/acbuf.h: Fix undeclared `uint' with GCC 5.4

GCC 5.4 complains that `uint' is unknown.  Plainly this syntactic
sugar has been removed.  Replacing with `unsigned int' fixes the
error.

Signed-off-by: Bob Ham <rah@settrans.net>
---
 include/acbuf.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/acbuf.h b/include/acbuf.h
index 33dbde8..231297e 100644
--- a/include/acbuf.h
+++ b/include/acbuf.h
@@ -55,14 +55,14 @@ class acbuf
          * \param maxlen Maximum amount of data to write
          * \return Number of written bytes, negative on failures, see write(2)
          */
-        int syswrite(int fd, unsigned int maxlen=MAX_VAL(uint));
+        int syswrite(int fd, unsigned int maxlen=MAX_VAL(unsigned int));
 
         /*
          * Reads from a file descriptor and append to buffered data, update position indexes.
          * \param fd File descriptor
          * \return Number of read bytes, negative on failures, see read(2)
          */
-        int sysread(int fd, unsigned int maxlen=MAX_VAL(uint));
+        int sysread(int fd, unsigned int maxlen=MAX_VAL(unsigned int));
 
 
     protected:
-- 
2.1.4

